src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id, completed, title, user,
+  } = todo;
+
+  return (
+    <article data-id={`${id}`} className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
Consider using the 'classnames' library to conditionally apply classes to the element. It makes the code cleaner and more readable. Example: import cx from 'classnames'; ... <article className={cx('TodoInfo', {'TodoInfo--completed': completed})}>